walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -4,6 +4,14 @@
   width: 650px;
 }
```

Comment:
Looks like maybe this file shouldn't be in this pull request? Looks like a tutorial css file, and I don't think it'll actually style anything on your website. Looks like there might be a trash icon somewhere where you can remove this file (https://github.blog/changelog/2018-07-31-removing-files-from-a-pull-request/)